sdk/keyvault/azure-security-keyvault-secrets/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## 4.3.0-beta.1 (Unreleased)
-
+## 4.2.2 (2020-10-08)
+- Updated versions for azure-core and azure-identity.
```

Comment:
follow the new style of changelog updates.
Add a section called Dependency updates and mention this under that.